backend/small_eod/event/migrations/0002_auto_20200103_1715.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.0.1 on 2020-01-03 17:15
```

Comment:
Ja bym proponował przed stworzeniem PR zrobić jednak z tego jedną migracje.

Reply:
Stare migracje usuniete - stworzone nowe. Czasem "makemigrations" robi pare plikow, chyba jest to zwiazane z relacjami